index.html

```diff
@@ -1,84 +1,111 @@
+  <link rel="stylesheet" href="normalize.css">
+  <link rel="stylesheet" href="skeleton.css">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="awesomplete.css">
+
+  <!-- Favicon
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+  <link rel="icon" href="favicon.ico" type="image/x-icon">
+
+</head>
+<body>
+
+  <!-- Primary Page Layout
+  –––––––––––––––––––––––––––––––––––––––––––––––––– -->
+    <header>
+        <button id="index" class="headerbrand">Tetick</button>
+        <button id="save" class="headerbutton">Save</button>
+        <button id="about-link" class="headerbutton">About</button>
+        <button id="wrong-link"class="headerbutton">Wrong</button>
+    </header>
+    <div class="swallow">
+        <div class="schedule">
+            <div class="nav">
+                    <span class="button navbutton stuffbutton" id="prev"> ← </span>
+                    <span class="navcounter" id="counter">0/?</span>
+                    <span class="button navbutton stuffbutton" id="next"> → </span>
+            </div>
+            <table>
+                <thead>
+                    <tr>
+                        <th>Monday</th>
+                        <th>Tuesday</th>
+                        <th>Wednesday</th>
+                        <th>Thursday</th>
+                        <th>Friday</th>
+                    </tr>
+                </thead>
+                <tbody>
+                    <tr>
+                        <td><span id="start_time">8:40</span><span id="end_time">17:30</span><div class="eatd" id="0"></div></td>
+                        <td><div class="eatd" id="1"></div></td>
+                        <td><div class="eatd" id="2"></div></td>
+                        <td><div class="eatd" id="3"></div></td>
+                        <td><div class="eatd" id="4"></div></td>
+                    </tr>
+                </tbody>
+            </table>
+            <div class="form">
+                <div class="dontfilldiv">
+            		day(1-5) <input class="short" id="d">, between <input class="short" id="s"> and <input class="short" id="e">
+            		<button class="dontfillbutton" id="dontfill"> Don't fill </button>
+                </div>
+                <div class="flashlight">
+                    <button class="stuffbutton" id="prevb">Prev</button>
+                    <button class="stuffbutton" id="fl">Flashlight</button>
+                    <button class="stuffbutton" id="nextb">Next</button>
+                </div>
+        		<div id="fdate" style="text-align: center; color: #aaa; font-size: 15px;">course data is from </div>
+                <div id="state">blank</div>
+            </div>
+        </div>
+        <div class="stuff">
+            <div class="options">
+                <div>Department: <input style="position:/*
+.eatd{position: relative; height: 100%; cursor: pointer;}*/relative; left:44px;" id="dept" class="" placeholder="EE, MINE etc."></div>
```

Comment:
no px-oriented and/or long inline styling please. most CSS frameworks have stuff for aligning forms.